packages/editor/src/js/viewmodel.js

```diff
@@ -47,8 +48,34 @@ function initializeEditor(content, blockDefs, thumbPathConverter, galleryUrl) {
     logoPath: 'rs/img/mosaico32.png',
     logoUrl: '.',
     logoAlt: 'mosaico',
+    toggleSaveBlockModal: ko.observable(null),
   };
 
+  function recursivelyUnwrapObservable(obj) {
```

Comment:
Is it possible to provide a global comment to quickly explain what observables are in knockout.js and why we need this function in the code.